src/AccessToken.php

```diff
@@ -421,7 +430,12 @@ private function loadPhpsecPublicKey(string $modulus, string $exponent): string
                 $exponent
             ]), 256),
         ]);
-        return $key->toString('PKCS8');
+        // Ensure the key is correctly initialized
+        $formattedPublicKey = $key->toString('PKCS8');
+        if (is_string($formattedPublicKey)) {
+            return $formattedPublicKey;
+        }
```

Comment:
Okay I was able to track it down to this change:

https://github.com/phpseclib/phpseclib/commit/f0194cf63959d1d5f49e583403f862a9e4c6900f

It seems like if you do `$key->toString('Raw')`, it will return an array. So this will never return an array for us, since we have a hardcoded value.

I think the best thing to do then would just be to add a string cast to get rid of the error. Something like this:
```suggestion
            return (string) $key->toString('PKCS8');
```

Since there seems to be multiple things happen in this PR, I'm going to open a new PR to solve the linting issue, and that part can be removed.

Reply:
Update - this still fails, so I am going to use the method here of throwing a type exception